lib/main.dart

```diff
@@ -1,65 +1,96 @@
 import 'package:flutter/material.dart';
 import 'package:get_it/get_it.dart';
-import 'package:wpi_campus/add_event.dart';
-import 'package:wpi_campus/ui/home.dart';
+import 'package:wpi_campus/ui/add_event.dart';
+import 'package:wpi_campus/model/home_event.dart';
+import 'package:wpi_campus/model/user_event.dart';
 import 'package:wpi_campus/model/repositories/data_repository.dart';
 import 'package:wpi_campus/model/repositories/firebase/firebase_data_repository.dart';
-import 'package:wpi_campus/ui/user_events.dart';
+import 'package:wpi_campus/ui/home_event_card.dart';
+import 'package:wpi_campus/ui/user_event_card.dart';
 
 final get = GetIt.instance;
 
+// todo consider using a material swatch instead of just primary color
+final _theme = ThemeData(primaryColor: Color.fromRGBO(172, 43, 55, 1));
+
 void main() {
   get.registerSingleton<DataRepository>(FirebaseDataRepository());
   runApp(MaterialApp(
     title: 'WPI Event Manager',
-    theme: ThemeData(primaryColor: Color.fromRGBO(172, 43, 55, 1)),
-    home: MainPage(),
+    theme: _theme.copyWith(brightness: Brightness.light),
+    darkTheme: _theme.copyWith(brightness: Brightness.dark),
+    themeMode: ThemeMode.system,
+    home: _MainPage(),
```

Comment:
That is a slick way of doing it. 

Would it be better having this in a settings page though?

Reply:
This should adopt the theme to be dark mode/light mode depending on the system, but if you want to do this in a settings page, you could also do that too